### DISCUSSION TITLE
Move react_ujs to es module flow

### DISCUSSION THREAD

**Discussion by honia19** [reactjs/react-rails#1344] [General]
I don't have any bugs but I caught a problem when we decided to move our app to module flow we have seen a problem with **require** especially in the server_render.js file. We are using mixed js and ts

```
var componentRequireContext = require.context('components', true);
var ReactRailsUJS = require('react_ujs');
ReactRailsUJS.useContext(componentRequireContext);
```

Is it possible to move this code to ES module approach? Who is faced with this issue?

**Comment by Judahmeek** [comment]
Google "import CommonJS module in ESM"
